pkg/microservice/systemconfig/core/codehost/repository/mongodb/codehost.go

```diff
@@ -85,6 +86,9 @@ func (c *CodehostColl) List(args *ListArgs) ([]*models.CodeHost, error) {
 	if args.Owner != "" {
 		query["namespace"] = args.Owner
 	}
+	if args.Source != "" {
```

Comment:
The namespace of gitlab is empty, so you need to confirm that the owner value of the caller is empty